action.yaml

```diff
@@ -16,6 +16,9 @@ inputs:
   issue-title:
     description: The custom subtitle for the issue
     required: false
+  allow-workflow-initiator-as-approver:
+    description: Whether or not to filter out the user who initiated the workflow as an approver if they are in the approvers list
```

Comment:
I could be overthinking this, but it might be clearer if the input was called `exclude-workflow-initiator-as-approver`. What are your thoughts?

Reply:
Yeah I agree with that. Just make sure that it defaults to false though so it doesn't break existing users!

Reply:
@trstringer Updated 😄